blank-canvas-blocks/sass/blocks/_button.scss

```diff
@@ -1,15 +1,17 @@
 /**
  * Button
  */
+
+// NOTE: These remain for the styling of buttons that are NOT blocks and is used elsewhere.  This can be removed when those no longer exist.
 @mixin button-main-styles {
-	font-weight: var(--wp--custom--button--font-weight);
-	font-family: var(--wp--custom--button--font-family);
-	font-size: var(--wp--custom--button--font-size);
-	line-height: var(--wp--custom--button--line-height);
-	border-radius: var(--wp--custom--button--border-radius);
+	font-weight: var(--wp--custom--button--typography--font-weight);
+	font-family: var(--wp--custom--button--typography--font-family);
+	font-size: var(--wp--custom--button--typography--font-size);
+	border-radius: var(--wp--custom--button--border--radius);
 	text-decoration: none; // Needed because link styles inside .entry-content add a text decoration
```

Comment:
I think this line now needs to be moved to the `.wp-block-button__link` selector so that buttons don't appear with an underline: 

<img width="719" alt="Screen Shot 2021-03-18 at 5 51 26 PM" src="https://user-images.githubusercontent.com/5375500/111702404-71c0cf80-87f9-11eb-9d8a-68ed8e803c27.png">

Reply:
Thanks.  Added to reg'ler block button stylin' too.